hw_6.py

```diff
@@ -0,0 +1,11 @@
+"""
+Write a program to find the longest word in the file.
+"""
+
+
+def long_word():
+    with open('file/file.txt', 'r') as f:
+        print(max(f.read().split(), key=len))
```

Comment:
1. Try to avoid `print` function in the functions. Use `return` instead.
2.  This line is too complicated. Please try to split it a little bit.

Reply:
Done